src/App.js

```diff
@@ -2,15 +2,59 @@ import React from 'react';
 import './App.scss';
 
 class App extends React.Component {
-  addOne = () => {};
+  state = {
+    count: 0,
+  }
+
+  addOne = () => {
+    this.setState(state => ({
```

Comment:
Краще явно вказувати, що це попердній стейт
```suggestion
    this.setState(prevState => ({
```